Makefile

```diff
@@ -39,7 +39,7 @@ test:
 
 .PHONY: lint
 lint:
-	golangci-lint run ./cmd/... ./pkg/... --timeout 7m
+	golangci-lint run ./cmd/... ./pkg/... --timeout 7m --max-issues-per-linter 0
```

Comment:
This flag should be set in the configuration file. https://golangci-lint.run/usage/configuration/

Reply:
Updated, thanks!